decentraland/unity-renderer
# fix: biw teleport and edit

## What does this PR change?

This PR unifies the data that builder in world and builder in world panel uses for the lands with access in the data store.

This way, we only need 1 call to get all the data and we can jump and edit without any problem

## How to test the changes?


1. Go to: https://play.decentraland.zone/index.html?renderer=urn:decentraland:off-chain:renderer-artifacts:fix/biw-jump-editor
2. Open the builder in world panel
3. Click on the editor button on the scenes or lands
4. Check that the builder in world is opened

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
